lib/vagrant-libvirt/config.rb

```diff
@@ -502,7 +502,11 @@ def finalize!
         @memory = 512 if @memory == UNSET_VALUE
         @cpus = 1 if @cpus == UNSET_VALUE
         @cpu_mode = 'host-model' if @cpu_mode == UNSET_VALUE
-        @cpu_model = 'qemu64' if @cpu_model == UNSET_VALUE
+        @cpu_model = if (@cpu_model == UNSET_VALUE and @cpu_mode == 'custom')
+            'qemu64'
```

Comment:
previous default value was qemu64
right now for keep same settings require define cpu_mode to "custom"
i think default settings and behaviour should not be changed.